PyInstaller/building/makespec.py

```diff
@@ -501,6 +510,9 @@ def main(scripts, name=None, onefile=None,
         # We need to encapsulate it into apostrofes.
         bundle_identifier = "'%s'" % bundle_identifier
 
+    if osx_app_console:
+        osx_app_console = True
+
```

Comment:
Isn't `osx_app_console` already a boolean? 

If it is possible for it to be of some other type that we need to convert, an explicit conversion would be nicer: `osx_app_console = bool(osx_app_console)`.

Reply:
yup, going to do the explicit conversion thanks!